lib/reverse_string.rb

```diff
@@ -1,5 +1,17 @@
 def reverse_characters(str)
+    reverse_characters_array = []
+        string_array = str.split("")
```

Comment:
Nice work! Lines 3-7 can all be moved to the left, so that line 3 is aligned with line 2, and line 5 is a space more to the right.